### PR TITLE
80872: Add ivc_champva_form model, factory and tests

### PR DESCRIPTION
**Pull request by balexandr**
## Summary
After creating the table [here](https://github.com/department-of-veterans-affairs/vets-api/pull/16367) we needed to create a basic model with some lightweight tests.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/80872

## Testing done

- [ ] *New code is covered by unit tests*

## What areas of the site does it impact?
None right now, new logic

## Acceptance criteria
Given that the system requires validation and tests for a new model,
When a IvcChampvaForms model is added,
Then I should be able to create, read, update, and delete products.

### CODE REVIEW

**Review by michaelclement — DISMISSED** [review]
lgtm

**Review by cloudmagic80 — APPROVED** [review]
LGTM. No glaring issues

File: app/models/ivc_champva_form.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class IvcChampvaForm < ApplicationRecord
+  validates :email, presence: true
+  validates :email, uniqueness: true
```
Comment (rmtolmach) [inline app/models/ivc_champva_form.rb:5]: Why not make it a one-liner? 
```rb
validates :email, presence: true, uniqueness: true
```

File: spec/models/ivc_champva_forms_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe IvcChampvaForm, type: :model do
```
Comment (rmtolmach) [inline spec/models/ivc_champva_forms_spec.rb:5]: What about some non-happy path tests like if the email is not present or unique?

**Review by rmtolmach — COMMENTED** [review]
please add some additional tests

File: spec/models/ivc_champva_forms_spec.rb (hunk 2 above)
Comment (balexandr) [reply spec/models/ivc_champva_forms_spec.rb:5]: Ah thanks, I knew I was forgetting something

**Review by rmtolmach — APPROVED** [review]
Thanks!

**Pull request merged**